tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php

```diff
@@ -3170,6 +3170,15 @@ public function provideCommentsCases()
 $a = 3; # 3
 $a = 4; /** 4 */
 echo 1;
+',
+        ];
+
+        yield 'exceptions' => [
+            '<?php
+new Dummy(/* a */);
+new Dummy(/** b */);
+foo(/* c */);
```

Comment:
what about `foo('x'/* c */, 'y');` ?

I think it still enforces space after `*/` which is againt every convention

Reply:
also for block comment before `->`, see https://github.com/atk4/data/pull/827/files#diff-1b13a61af17b481a4e15159a87f2a89937836e504284460f2164bfafc1d32348R83